packages/widgets/src/docklayout.ts

```diff
@@ -2193,7 +2193,7 @@ namespace Private {
     renderer: DockLayout.IRenderer
   ): TabLayoutNode {
     // Create the tab bar for the layout node.
-    let tabBar = renderer.createTabBar(this._document);
+    let tabBar = renderer.createTabBar();
```

Comment:
Would it be better to be able to pass an optional document to `realizeTabAreaConfig` to be forward to `createTabBar`?

Reply:
Maybe - I am struggling to see where its used (as it doesn't seem to be hit in the tests and the old version of TypeScript isn't helpful).
I will have more time to dig into it in the morning, if your not looking for a quick fix today,

Reply:
@fcollonval:  I changed the code as you suggested and did a bit more sanity checking...

Also I added TabPanel support to the web-components:  https://gordonsmith.github.io/hpcc-js-wc/src/layouts/lumino/tabPanel.html